docs/aurora/running-jobs-aurora.md

```diff
@@ -72,7 +72,9 @@ More information on the PBS options above, as well as other PBS options, can be
 
 ## Working Around Node Failures
 
-As Aurora is still a pre-production supercomputer, node failures are a fact of life. If you would like to increase the chances that a large job does not terminate due to a node failure, you may choose to interactively route your MPI job around nodes that fail during your run. To do this, you must run interactively and use must manually adjust your run on the fly to remove nodes that have been marked as failed.
+As Aurora is in early production stage, node failures are a fact of life. If you would like to increase the chances that a large job does not terminate due to a node failure, you may choose to interactively route your MPI job around nodes that fail during your run. To do this, you must run interactively and use must manually adjust your run on the fly to remove nodes that have been marked as failed.
```

Comment:
do we have an example and/or more detailed instructions for how to accomplish that somewhere in the docs? I seem to recall it being somewhere (@martagarciamartinez might have mentioned it recently?), but all I can find is https://docs.alcf.anl.gov/aurora/running-jobs-aurora/#controlling-where-your-job-runs

if we do, would be good to link here